### PR TITLE
Add encode setup time, increase timer, remove spaces for functions

### PR DESCRIPTION
**Pull request by KaptenJansson**
* Increase test duration to 8 (was 5) seconds to allow Mac enough time to start capture.
* Increase stats gathering frequency to 100 (was 1000) ms.
* Removed spaces in functions before ( braces.
* Report encode setup time, basically how long it takes from start gathering stats until getStats reports back non zero input frame rate.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
There were some JSCS errors about callbacks not being defined and local variables being declared several times in for in loops.

File: samples/web/content/testrtc/js/camresolutionstest.js [hunk 1]
```diff
@@ -91,11 +91,12 @@ CamResolutionsTest.prototype = {
     var call = new Call();
     call.pc1.addStream(stream);
     call.establishConnection();
-    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 1000);
-    setTimeoutWithProgressBar( function() {
+    this.collectStatsStartTime = Date.now();
```
Comment (KaptenJansson) [inline samples/web/content/testrtc/js/camresolutionstest.js]: Not sure if this is good enough, should I leave it or move it to call.gatherStats?

Comment (andresusanopinto) [inline samples/web/content/testrtc/js/camresolutionstest.js]: I have no strong preference here. Btw the travis check is failing.

Comment (KaptenJansson) [inline samples/web/content/testrtc/js/camresolutionstest.js]: I know, but it's complaining about stuff I haven't changed as well. New grunt criteria might have been added as well, will sort it out.

**Comment by KaptenJansson** [comment]
I've merged master and the soon to land pull request for issue #412 . So once pull request for #412 lands  the commits will be squashed right? Leaving the merge and Fixed grunt errors commits?

**Comment by KaptenJansson** [comment]
Basically do nothing until #412 is on Master.

File: samples/web/content/testrtc/js/call.js [hunk 2]
```diff
@@ -48,15 +48,19 @@ Call.prototype = {
...
+  // TODO (jansson) expand to cover audio as well.
+  gatherStats: function(peerConnection, statsCb, firstFrameCb, interval) {
+    var firstFrameCbSent = false;
     var stats = [];
     getStats_();
 
     function getStats_() {
       if (peerConnection.signalingState === 'closed') {
-        callback(stats);
+        statsCb(stats);
```
Comment (KaptenJansson) [inline samples/web/content/testrtc/js/call.js]: I shortened the callback names due to grunt (jshint and JSCS) cannot handle function definitions with linebreaks.

**Comment by KaptenJansson** [comment]
#412 is now closed hence a merge can be done.

File: samples/web/content/testrtc/js/call.js [hunk 3]
```diff
@@ -48,15 +48,19 @@ Call.prototype = {
     this.constrainOfferToRemoveVideoFec_ = true;
   },
 
-  // When the peerConnection is closed the callback is called once returning
-  // with an array of gathered stats.
-  gatherStats: function(peerConnection, callback, interval) {
+  // When the peerConnection is closed the statsFinishedCallback is called once
+  // returning with an array of gathered stats. timeForFirstFrameCallback will
+  // be called once when googFrameRateInput is > 0 with a timestamp in
```
Comment (andresusanopinto) [inline samples/web/content/testrtc/js/call.js]: Please update the documentation and fix "firstFrameCb will be called at most once when..."

File: samples/web/content/testrtc/js/call.js (hunk 2 above)
Comment (andresusanopinto) [inline samples/web/content/testrtc/js/call.js]: -_-'

File: samples/web/content/testrtc/js/call.js (hunk 3 above)
Comment (KaptenJansson) [inline samples/web/content/testrtc/js/call.js]: Done!

**Comment by KaptenJansson** [comment]
PTAL

**Comment by KaptenJansson** [comment]
Ping

**Pull request merged**